src/common/transformations/src/ov_ops/placeholder.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ov_ops/placeholder.hpp"
+
+namespace ov {
+namespace op {
+namespace internal {
+
+Placeholder::Placeholder() : ov::op::Op() {
+    validate_and_infer_types();
+}
+
+void Placeholder::validate_and_infer_types() {
+    set_output_type(0, ov::element::undefined, ov::PartialShape{});
```

Comment:
```suggestion
    set_output_type(0, ov::element::undefined, ov::PartialShape::dynamic());
```
To be any shape not scalar?

Reply:
PlaceHolder has been removed

Reply:
PlaceHolder has been removed